server/routes/taskList.js

```diff
@@ -35,13 +37,20 @@ module.exports = ({ prisonerService, licenceService, audit }) => router => {
       const licenceStatus = getLicenceStatus(licence)
       const allowedTransition = getAllowedTransition(licenceStatus, req.user.role)
       const { statusLabel } = getStatusLabel(licenceStatus, req.user.role)
+      let errors
+
+      if (licenceStatus.stage === 'ELIGIBILITY' && req.user.role === 'CA') {
+        const errorCodesList = await caService.getReasonForNotContinuing(bookingId, res.locals.token)
+        errors = errorCodesList.length > 0 ? errorCodesList.map(error => taskListErrors[error]) : null
```

Comment:
The benefit of relying on empty array rather than null is that you don't need to handle the null here - just pass an empty array through. We're modelling no errors as empty array rather than null.

Reply:
done